examples/step-71/doc/intro.dox

```diff
@@ -0,0 +1,508 @@
+been replaced with interior penalty techniques. To derive this method,
+we consider a $C^0$ shape function $v_h$ which vanishes on
+$\partial\Omega$. Since the higher order derivatives of $v_h$ have two
+values on each interface $e\in \mathbb{F}$ (shared by the two cells
+$K_{+},K_{-} \in \mathbb{T}$), we cope with this discontinuity by
+defining the following single-valued functions on $e$:
+@f{align*}{
+  \jump{\frac{\partial^k v_h}{\partial \mathbf n^k}}
+  &=
+  \frac{\partial^k v_h|_{K_+}}{\partial \mathbf n^k} \bigg |_e
+  - \frac{\partial^k v_h|_{K_-}}{\partial \mathbf n^k} \bigg |_e,
+  \\
+  \average{\frac{\partial^k v_h}{\partial \mathbf n^k}}
+  &=
+  \frac{1}{2}
+  \bigg( \frac{\partial^k v_h|_{K_+}}{\partial \mathbf n^k} \bigg |_e
+  + \frac{\partial^k v_h|_{K_-}}{\partial \mathbf n^k} \bigg |_e \bigg )
+@f}
+for $k =1,2$ (i.e., for the gradient and the matrix of second
+derivatives), and where $\mathbf n$ denotes a unit vector normal to
+$e$ pointing from $K_+$ to $K_-$. In the
+literature, these functions are referred to as the "jump" and
+"average" operations, respectively.
+
+To obtain the $C^0$ IP approximation $u_h$, we left multiply the
+biharmonic equation by $v_h$, and then integrate over $\Omega$. As
+explained above, we can't do the integration by parts on all of
+$\Omega$ with these shape functions, but we can do it on each cell
+individually since the shape functions are just polynomials on each
+cell. Consequently, we start by using the following
+integration-by-parts formula on each mesh cell $K \in {\mathbb{T}}$:
+@f{align*}{
+  \int_K v_h (\Delta^2 w_h)
+  &= \int_K v_h (\nabla\cdot\nabla) (\Delta w_h)
+  \\
+  &= -\int_K \nabla v_h \cdot (\nabla \Delta w_h)
+     +\int_{\partial K} v_h (\nabla \Delta w_h \cdot \mathbf n).
+@f}
+At this point, we have two options: We can integrate the domain term's
+$\nabla\Delta w_h$ one more time to obtain
+@f{align*}{
+  \int_K v_h (\Delta^2 w_h)
+  &= \int_K (\Delta v_h) (\Delta w_h)
+     +\int_{\partial K} v_h (\nabla \Delta w_h \cdot \mathbf n)
+     -\int_{\partial K} (\nabla v_h \cdot \mathbf n) \Delta w_h.
+@f}
+For a variety of reasons, this turns out to be a variation that is not
+useful for our purposes.
+
+Instead, what we do is recognize that
+$\nabla\Delta w_h = \text{grad}\,(\text{div}\,\text{grad}\, w_h)$, and we
+can re-sort these operations as
+$\nabla\Delta w_h = \text{div}\,(\text{grad}\,\text{grad}\, w_h)$ where we
+typically write $\text{grad}\,\text{grad}\, w_h = D^2 w_h$ to indicate
+that this is the "Hessian" matrix of second derivatives. With this
+re-ordering, we can now integrate the divergence, rather than the
+gradient operator, and we get the following instead:
+@f{align*}{
+  \int_K v_h (\Delta^2 w_h)
+  &= \int_K (\nabla \nabla v_h) : (\nabla \nabla w_h)
+     +\int_{\partial K} v_h (\nabla \Delta w_h \cdot \mathbf n)
+     -\int_{\partial K} (\nabla v_h \otimes \mathbf n) : (\nabla\nabla w_h)
```

Comment:
I might be being dense but I'm not sure what `otimes` is supposed to mean here. We define `:` below so I think we should also define this operator.

Reply:
It's the outer product. I fixed that.